KarhooSDK/Network/HttpClient/JsonHttpClient/JsonHttpClient.swift

```diff
@@ -131,6 +133,13 @@ private extension JsonHttpClient {
     func absoluteUrl(endpoint: APIEndpoint) -> URL {
         let environmentDetails = KarhooEnvironmentDetails(environment: Karhoo.configuration.environment())
 
+        guard endpoint != .vehicleRules else {
```

Comment:
let's have this guard before the environment details retrieval in order to provide a better separation